src/build.rs

```diff
@@ -77,21 +83,22 @@ impl Builder {
                     continue;
                 }
 
-                if name.starts_with("lib") {
-                    name = &name[3..];
-                }
-
                 let hyphen = name.find('-');
                 let hyphen = match hyphen {
                     Some(h) => h,
                     None => continue,
                 };
                 let name = &name[..hyphen];
+                let match_name = if name.starts_with("lib") {
+                    &name[3..]
+                } else {
+                    &name
+                };
```

Comment:
Why change to the more verbose idiom here?

Reply:
I'm not clear if moving the "lib" check is significant - what is the effect?

Reply:
Because we need `&name` still to bucket `libX` and `X` seperately (unless that's not what you meant)

Reply:
OK, thanks